### DISCUSSION TITLE
Any idea what cAr, cAb, and cAg are?

### DISCUSSION THREAD

**Discussion by Trainmaster9977** [ScanMountGoat/Smush-Material-Research#38] [Q&A]
Looking through shader code/metadata and stumbled across that.

**Comment by ScanMountGoat** [comment]
I haven't looked into those values personally. You should be able to use the metadata to figure out which values in the shader code to edit in RenderDoc. If I had to guess, it sounds like something related to color grading. It should be a lot clearer what they do after hard coding different RGB values in RenderDoc's shader editor like (0,0,0), (1,1,1), (1,0,0), (5,5,5), etc.
